recipes-connectivity/networkmanager/networkmanager-1.4.4/update-routing-tables.patch

```diff
@@ -0,0 +1,128 @@
+#OpenXT:Update the routing tables for different bridges as setup by network-slave.
+#       These routing tables are reffered when routing packets for different networks.
+--- a/src/nm-policy.c
++++ b/src/nm-policy.c
+@@ -472,8 +472,10 @@ update_ip4_routing (NMPolicy *self, gboo
+ 	}
+ 	g_assert ((best || vpn) && best_ac);
+ 
++#if 0
```

Comment:
Should be removed by the patch instead of defined out.

Reply:
Ack, will remove it in the patch.